docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
   web:
     build: .
     environment:
-      ENV_NAME: production
```

Comment:
Could this now be `production`? As we've introduced the `IS_LOCAL_DOCKER_ENV` variable anyways I think it would work?

Reply:
It could be but I would need to change the settings.yml config to ` production: <%= ENV.fetch("DEV_AUTH_ENABLED", false) %>` which i was bit reluctant to do